### ISSUE TITLE
NTR: Transepithelial nasal potential difference abnormal (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8002]
New term request Transepithelial nasal potential difference abnormal (elthat_100624033553): NPD; raised (more negative)  UMLS: 2751565

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000100
- label: Hyperpolarized transepithelial nasal potential difference
- parents: [Abnormal respiratory system physiology (HP:0002795)](https://hpo.jax.org/app/browse/term/HP:0002795)
- pmid: PMID:21594779
- def: Increased (mopre negative) voltage across the nasal epithelium.
- comment: The resulting (or basal) potential difference (PD) is negative or polarized (viewed from the epithelial surface) and in normal subjects, it is usually between -15 and -25 mV. The measurement continues by the sequential perfusion of compounds that block inwardly directed sodium transport through inhibition of ENaC (amiloride), followed by augmentation of chloride transport through CFTR. During perfusion of amiloride, the potential difference depolarizes as ENaC transport is reduced, and the PD typically approaches a low polarizing value (typically between 0 and -10 mV).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8002
- synonyms:

**Comment by MickeySegal** [comment]
Typo in def: **mopre**

**Comment by pnrobinson** [comment]
fixed, thanks!
